content/en/vendor/disclaimer.md

```diff
@@ -0,0 +1,12 @@
+<!-- 
+---
+title: "Disclaimer"
+---
+--> 
+
+This page includes content about running Tekton with specific platforms and
+cloud providers. The accuracy and freshness of this vendor documentation may
```

Comment:
nit: suggest "varies by vendor" rather than "may vary".

Should disclaimer have a statement like: "Please direct issues to the specific vendor rather than to the Tekton community at large" or some such?

To that end, perhaps a prerequisite for adding a vendor doc is to provide appropriate vendor contact information for reporting issues or seeking help and clarification?

Reply:
> Should disclaimer have a statement like: "Please direct issues to the specific vendor rather than to the Tekton community at large" or some such?
> 
> To that end, perhaps a prerequisite for adding a vendor doc is to provide appropriate vendor contact information for reporting issues or seeking help and clarification?

I think this would implicitly restrict contributions to vendors only. If a member of the community wants to cover a particular use case involving specific vendors, I think they should be able to do so, in the spirit of OSS.